app/models/solr_submission.rb

```diff
@@ -44,13 +42,25 @@ def field_semantics
         access_level: 'access_level_ss',
         semester: 'semester_ssi',
         abstract: 'abstract_tesi',
-        defended_at: 'defended_at_dtsi',
-        released_metadata_at: 'released_metadata_at_dtsi',
+        defended_at_dtsi: 'defended_at_dtsi',
+        released_metadata_at_dtsi: 'released_metadata_at_dtsi',
         degree_name: ['degree_name_ssi'],
         degree_description: ['degree_description_ssi']
       }
     end
 
+    def final_submission_files_uploaded_at_dtsi
```

Comment:
i attempted to run this with production data, and it appears that we have some where this field is nil, we should probably account for this somehow, maybe doing  an `|| nil` would be good enough? 


```NoMethodError: undefined method `strftime' for nil:NilClass```

Reply:
Yeah that should work.  Also, I think I need to convert to UTC.